snuba/utils/streams/topics.py

```diff
@@ -25,12 +25,14 @@ class Topic(Enum):
     SUBSCRIPTION_RESULTS_SESSIONS = "sessions-subscription-results"
     SUBSCRIPTION_RESULTS_METRICS = "metrics-subscription-results"
     QUERYLOG = "snuba-queries"
+    PROFILES = "processed-profiles"
```

Comment:
No one yet, I'm making that PR right now.

Reply:
Ok. Random suggestion but how about calling it `profiles` instead of `processed-profiles` since we don't use that prefix elsewhere. Not that we really have any conventions about naming here but in some other cases I believe `ingest-topicname` usually means unprocessed and `topicname` means processed.

Reply:
Ah, I assumed we were going to call that earlier one `ingest-profiles`. In that case this seems fine to me.